CommonLibSF/include/RE/A/Actor.h

```diff
@@ -26,7 +26,17 @@ namespace RE
 		inline static bool IsPlayerCharacterInChargen()
 		{
 			auto PC = PlayerCharacter();
-			return *(stl::adjust_pointer<bool>(PC, 0xF24));
+
+            std::uint8_t flags = *(RE::stl::adjust_pointer<std::uint8_t>(PC, 0xF24));
+
+            if (flags & 1 || flags & 2 || flags & 4)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
```

Comment:
I'd add a comment about this behaviour here and let someone else double-check if that's indeed what's happening. We shouldn't be making such changes willy-nilly until at least those flags are decoded.

Reply:
Sure. For anybody's information -  prior to PR I tested flags by calling papyrus function SetInChargen(Bool abDisableSaving, Bool abDisableWaiting, Bool abShowControlsDisabledMessage) (29FAA60) with each argument and flags were set correctly by it.